src/frontend/src/flows/authorize/index.ts

```diff
@@ -38,7 +38,7 @@ export const authnTemplateAuthorize = ({
         })
       : undefined;
 
-  const wrap = (title: string | TemplateResult) => html`
+  const wrap = (title: DynamicKey) => html`
```

Comment:
correct? it used to be a `string | TemplateResult` so I would have expected `TemplateElement`

Reply:
here it doesn't matter since it's a `DynamicKey`, which is more restrictive anyway! if we need a `string` or `TemplateResult` in the future we can change for a `TemplateElement`. Good catch though :)